nym-wallet/src/components/Buy/Tutorial.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { useState } from 'react';
+import { Button, Stack, Typography, Grid, Link } from '@mui/material';
+import { Tune as TuneIcon, BorderColor as BorderColorIcon, Paid as PaidIcon } from '@mui/icons-material';
+import { NymCard } from '../NymCard';
+import { SignMessageModal } from './SignMessageModal';
+
+// TODO retrieve this value from env
+const EXCHANGE_URL = 'https://buy.nymtech.net';
+
+const borderColor = 'rgba(141, 147, 153, 0.2)';
+
+const TutorialStep = ({
+  step,
+  title,
+  text,
+  icon,
+  divider,
+}: {
+  step: number;
+  title: string;
+  text: string | React.ReactNode;
+  icon: React.ReactNode;
+  divider?: boolean;
+}) => (
+  <Grid
+    item
+    md={4}
+    pb={2}
+    pr={1}
+    sx={{
+      borderRight: divider ? `1px solid ${borderColor}` : null,
+    }}
+  >
+    <Stack gap={2}>
+      <Stack direction="row" gap={1} alignItems="center">
+        {icon}
+        <Typography fontWeight={600} fontSize="12px">
+          {`STEP ${step}`}
+        </Typography>
+      </Stack>
+      <Typography fontWeight={600} variant="h6">
+        {title}
+      </Typography>
+      {typeof text === 'string' ? (
+        <Typography sx={{ color: (t) => t.palette.nym.text.muted }}>{text}</Typography>
+      ) : (
+        text
+      )}
```

Comment:
If `typeof text === 'string'` then you display it inside of `<Typography />`. 
Why not have text always be a `ReactNode` and pass in 
`<Typography sx={{ color: (t) => t.palette.nym.text.muted }}>{text}</Typography>` ?
Then here just use `{ text }` to display it here.